bravado/mapping/response.py

```diff
@@ -0,0 +1,30 @@
+class ResponseLike(object):
+    """
+    Define a common interface for bravado to interface with client side
+    response objects from various 3rd party libraries.
+
+    Subclasses are responsible for providing attrs for __required_attrs__.
+    """
```

Comment:
Well, this class is meant to be subclassed and complain in a helpful way when the subclass' implementation is not fully compliant.